radixdlt-core/radixdlt/src/test/java/com/radixdlt/sync/SometimesByzantineCommittedReader.java

```diff
@@ -177,6 +179,7 @@ public VerifiedTxnsAndProof build() {
 		}
 	}
 
+	@Ignore
```

Comment:
Could you please clarify why this is necessary?

Reply:
For some reason, a build on CI pipeline included this class into the list of tests and since the class is not test and not intended for such a use, the test failed and failed the whole build. Addition of this annotation solved the issue.